zkevm-circuits/src/witness/block.rs

```diff
@@ -566,6 +614,48 @@ pub fn block_convert(
         log::error!("withdraw root is not available");
     }
 
+    let bytecodes: BTreeMap<Word, Bytecode> = code_db
```

Comment:
make a standalone function for these

Reply:
added helper `get_bytecodes` `get_bytecode_map` to extract these bytecodes info.